ui/src/main/resources/CKEditor/OfficeImporter.xml

```diff
@@ -57,7 +57,7 @@
 {{/velocity}}
 
 {{velocity}}
-#if ("$!request.fileName" == '')
+#if ("$!request.fileName" == '' &amp;&amp; "$!xcontext.getAction()" == 'get')
```

Comment:
Why do you need the silent notation? And ``getAction()`` returns a string so it can be written as simple as:

```suggestion
#if ("$!request.fileName" == '' &amp;&amp; $xcontext.action == 'get')
```

Reply:
Indeed the silent notation is useless. Now regarding `getAction()`, I don't really understand why I can use `$xcontext.action`: Velocity makes a direct binding to the correct getter in that case? What's the advantage compared to using the defined getter?

Reply:
See http://velocity.apache.org/engine/1.7/vtl-reference.html#methods . It's a feature of Velocity. It makes the code more concise. We use this everywhere in our wiki pages and in our Velocity templates.

Reply:
Yeah I saw that. Wasn't sure about its usage. Thanks for the pointer.